src/date-range-picker.ts

```diff
@@ -1,3 +1,3 @@
-import { DateRangePicker } from './src/DateRangePicker.js';
+import { DateRangePicker } from './DateRangePicker.js';
 
 window.customElements.define('date-range-picker', DateRangePicker);
```

Comment:
I think you can just do this directly in `DateRangePicker.ts`.

Reply:
I tried this but was not able to get it to work reliably with the snowpack build. Also, I am now kind of decoupling the class and the use of the class in a component.